plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergMetastoreAccessOperations.java

```diff
@@ -83,7 +83,7 @@ public void testCreateTable()
                 ImmutableMultiset.builder()
                         .add(CREATE_TABLE)
                         .add(GET_DATABASE)
-                        .addCopies(GET_TABLE, 2)
+                        .add(GET_TABLE)
```

Comment:
In another PR i was suggested to use only `addCopies` and replace add with `addCopies(...,1)`.
I see you did the opposite here ;) For me `add`  is fine but just leaving this comment here to be consistent

Reply:
I think it was about consistency of neighbouring lines.

i do hope we will bring those counts to 1 (if by nothing else, then with some metastore cache)
and then we will have `add(element)` consistently.

I do not know whether it that other PR count of 1 is the expected value, so maybe there is a difference.